src/taoensso/sente.cljc

```diff
@@ -1339,15 +1341,18 @@
      (let [packer (coerce-packer packer)
 
            win-loc (enc/get-win-loc)
-           win-protocol  (:protocol win-loc)
+           protocol (case (keyword protocol)
```

Comment:
Probably good to add an assert if `win-loc` is nil and `protocol` not provided, and in the error explain that `protocol` must be provided.